celesta-maven-plugin/src/test/java/ru/curs/celesta/plugin/maven/AbstractCelestaMojoTestCase.java

```diff
@@ -72,22 +72,30 @@ void assertGeneratedCursors(String generatedSourcesDir, List<String> cursorPaths
 
         JavaCompiler javaCompiler = ToolProvider.getSystemJavaCompiler();
 
-        cursorPaths.forEach(p -> {
-            File f = new File(prefix, p);
-            assertTrue(f.exists());
-            assertEquals(0, javaCompiler.run(null, null, null, f.getPath()));
-
-            File expectedF = new File(expectedPrefix, p);
-            try {
-                assertEquals(
-                        StringUtils.normalizeSpace(FileUtils.readFileToString(expectedF)),
-                        StringUtils.normalizeSpace(FileUtils.readFileToString(f))
-                );
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        });
+        cursorPaths.forEach(p -> assertGeneratedCursor(p, prefix, expectedPrefix, javaCompiler));
+    }
+
+    void assertGeneratedCursor(String p, File prefix, File expectedPrefix, JavaCompiler javaCompiler) {
+        File f = new File(prefix, p);
+        assertTrue(f.exists());
+        assertEquals(0, javaCompiler.run(null, null, null, f.getPath()));
+
+        File expectedF = new File(expectedPrefix, p);
+        System.out.printf("Comparing files:\n  %s\n  %s\n", expectedF, f);
+        try {
+            assertEquals(
+                    replaceDateInGeneratedAnnotation(StringUtils.normalizeSpace(FileUtils.readFileToString(expectedF))),
+                    replaceDateInGeneratedAnnotation(StringUtils.normalizeSpace(FileUtils.readFileToString(f)))
+            );
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
 
+    String replaceDateInGeneratedAnnotation(String s) {
+        return s.replaceAll(
+                "@Generated\\( value = \"ru.curs.celesta.plugin.maven.CursorGenerator\", date = \".+\" \\)",
```

Comment:
I generally avoid using `.` (dot) in regexps because of its performance issues (see [this talk](https://www.youtube.com/watch?v=pLKrKFy1t1Y) for example). Non an issue here, probably, but `"[^"]+"` is way more explicit and effective. Maybe worth change it since we need a new commit anyway.

Reply:
Great catch! Addressed!